doc/doom_nvim.txt

```diff
@@ -32,7 +32,7 @@
      1. Layout................................|doom_nvim_configuration_layout|
      2. File Explorer................|doom_nvim_configuration_layout_explorer|
      3. Main Buffer................|doom_nvim_configuration_layout_mainbuffer|
-     4. Tagbar.........................|doom_nvim_configuration_layout_tagbar|
+     4. Symbols-Outline.........................|doom_nvim_configuration_layout_outline|
```

Comment:
The number of dots should be reduced here so that the line has the same length as the others

Reply:
Yes, you are right.

I fixed it on my side already. However, as I'm more of a GitLab-er, I'm wondering why the PR doesn't reflect the state of my branch...

Reply:
OK, I think I have it now. GitHub did, and I didn't delete the commit from `develop` after having cherry-picked it over to `fix/tagbar_confusion`. I'll do it right now. Please see https://github.com/NTBBloodbath/doom-nvim/pull/12 - the dots there should be fine. :relaxed: